internal/jobs/twitter.go

```diff
@@ -841,13 +920,12 @@ func (ts *TwitterScraper) GetProfileByID(baseDir, userID string) (*twitterscrape
 		return nil, err
 	}
 
-	ts.statsCollector.Add(stats.TwitterProfiles, 1)
+	ts.statsCollector.Add(stats.TwitterOther, 1)
```

Comment:
hmmm... why? This will retrieve 1 profile, so it should be `Twitter Profile`, no?